<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <h1>Jack Turnbull - Full Stack Developer (Scientific Automation)</h1>
    <p>Newcastle upon Tyne, United Kingdom | 07402895493 | jackturnbull21@gmail.com</p>
    <hr />

    <h2>Profile</h2>
    <p>
        Bioinformatics engineer with 3 years of experience turning complex biological data into actionable insights 
        that accelerate research and decision-making. Proven track record of building scalable tools and pipelines 
        that improve data quality, reduce processing time, and enable reproducible science. Skilled at bridging 
        biology, data engineering, and software development to deliver solutions that drive measurable impact.
    </p>
    <hr />

    <h2>Professional Experience</h2>
    <h3>Jul 2024 - Present | Bioinformatics Engineer, QuantuMDx | Newcastle upon Tyne</h3>
    <p>
        Spearheading the automation of multiplex qPCR assay design with scalable bioinformatics pipelines that 
        transform manual workflows into efficient, reproducible processes.
    </p>
    <ul>
        <li>
            Collaborated with research teams to identify workflow bottlenecks and automation opportunities
        </li>
        <li>
            Developed clean, well-documented code with thorough profiling and testing frameworks to ensure 
            efficient, maintainable, and high-performance pipelines that scale effectively
        </li>
        <li>
            Designed and implemented intuitive front-end interfaces for pipelines, improving accessibility by 
            allowing researchers to run automated analyses without coding experience
        </li>
        <li>
            Engineered automated pipelines for medical device log parsing and performance monitoring, 
            streamlining troubleshooting and driving clinical study milestones on schedule
        </li>
        <li>
            Generated and presented complex data insights to senior leadership and external collaborators, 
            guiding crucial decision-making in diagnostic development programs
        </li>
    </ul>
    <br />

    <h3>Apr 2023 - Mar 2024 | Biomedical Data Engineer, A4PBio | Newcastle upon Tyne</h3>
    <p>
        Delivered high-quality, privacy-compliant data at scale to power AI-driven digital pathology solutions for 
        personalised healthcare.
    </p>
    <ul>
        <li>
            Engineered robust ETL/ELT pipelines for multi-modal data, reducing manual data processing time 
            and accelerating data availability for algorithm development
        </li>
        <li>
            Standardised pathology images and patient metadata from RCT and RWE datasets to SDTM and 
            OMOP, ensuring regulatory-ready, ISO-compliant data products
        </li>
        <li>
            Safeguarded sensitive patient data through GDPR-compliant anonymisation and strict access 
            management, maintaining zero compliance incidents
        </li>
        <li>
            Optimised and maintained existing data curation and anonymisation scripts, leveraging Git and Jira 
            to support efficient, collaborative development
        </li>
        <li>
            Managed cross-team and external stakeholder communication to ensure data deliveries met deadlines 
            and contractual requirements
        </li>
        <li>
            Implemented FAIR data principles and DAMA-DMBOK practices, earning the FAIR Data Prize for 
            enhancing data discoverability and reuse across the organisation
        </li>
    </ul>
    <br />

    <h3>Mar 2022 - Sep 2022 | Bioinformatician, Wolfson Centre | Newcastle upon Tyne</h3>
    <p>
        Developed large-scale ChIP-Seq workflows to profile chromatin states and identify epigenetic mechanisms 
        regulating cell cycle genes in cancer.
    </p>
    <ul>
        <li>
            Built scalable ChIP-Seq pipelines for QC (FastQC), trimming (Cutadapt), alignment (Bowtie), and 
            peak calling (MACS), handling large-scale datasets efficiently
        </li>
        <li>
             Converted peak coordinates into 200bp-binned chromatin-state matrices, encoding combinations of 
            histone modifications to define active and repressive regulatory elements
        </li>
        <li>
            Integrated chromatin-state matrices with promoter capture Hi-C and gene annotations to map 
            regulatory interactions controlling cell cycle genes
        </li>
        <li>
            Applied unsupervised analyses to distinguish healthy vs. tumor samples, uncovering epigenetic 
            dysregulation driving cancer-related chromatin remodeling
        </li>
    </ul>
    <hr />

    <h2>Education</h2>

    <h3>
        Sep 2021 - Sep 2022 | Bioinformatics MSc, Newcastle University | Newcastle upon Tyne
    </h3>
    <ul>
        <li>Graduated with Distinction</li>
    </ul>

    <h3>
        Sep 2018 - Jun 2021 | Biomedical Sciences BSc Honours, Newcastle University | Newcastle upon Tyne
    </h3>
    <ul>
        <li>Graduated with First Class Honours</li>
    </ul>
    <hr />

    <h2>Technical Skills</h2>
    <ul>
        <li>
            Programming & Scripting: Python, R, Bash, SQL, SAS
        </li>
        <li>
            Web Development: HTML, CSS, JavaScript, Django, Flask, Dash, Streamlit
        </li>
        <li>
            Machine Learning: Scikit-learn, Tensorflow, Keras, PyTorch
        </li>
        <li>
            Data Manipulation & Visualization: Pandas, NumPy, Seaborn, Matplotlib, Plotly
        </li>
        <li>
            Data Engineering: Azure Data Factory / Databricks, Apache Spark, PostgreSQL, SQLite
        </li>
        <li>
            Cloud & DevOps: AWS, Azure, Docker, Kubernetes, Git, CI/CD
        </li>
    </ul>
    <hr />

</body>
<footer>
    <p>Last Updated: 03/09/2025</p>
</footer>
</html>